internal/resolution/variablesources/required_package_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("RequiredPackageVariableSource", func() {
 	It("should filter by version range", func() {
 		// recreate source with version range option
 		var err error
-		rpvs, err = variablesources.NewRequiredPackageVariableSource(packageName, variablesources.InVersionRange(">=1.0.0 !2.0.0 <3.0.0"))
+		rpvs, err = variablesources.NewRequiredPackageVariableSource(packageName, variablesources.InVersionRange(">=1.0.0 !=2.0.0 <3.0.0"))
```

Comment:
Masterminds/semver does not  support the `!` operator, instead it supports `!=`. This was a bit unexpected.

Reply:
What is the blast radius of this impact? Operator spec.version field? FBC metadata?

Reply:
(Note this is test code.)
It _should_ only impact the **operator.spec.Version**, which previously only supported a single semver, not a range. But it is something to be aware of. Possibly something to port to Masterminds/semver if they're interested.